### PR TITLE
Add host display properties (e.g., color, font size) to add/edit host view.

### PR DESCRIPTION
**Pull request by khorimoto**
Note that the XML had 2-space tabs for some reason, so I fixed that as well. That's why it shows up as a bigger XML change than it actually was.

### CODE REVIEW

**Comment by jklein24** [comment]
Do you have a mock or spec that you're working off of for this view?

**Comment by khorimoto** [comment]
Nope. @bengolds told me just to add a basic outline and he will give me a spec/mock at a later point. The UI is rough and unpolished for now.

File: app/src/main/java/org/connectbot/HostEditorFragment.java [hunk 1]
```diff
@@ -234,6 +294,10 @@ public void onAttach(Context context) {
 		} catch (ClassCastException e) {
 			throw new ClassCastException(context.toString() + " must implement Listener");
 		}
+
+		// Now that the fragment is attached to an Activity, fetch the array from the attached
+		// Activity's resources.
+		mColorValues = getResources().obtainTypedArray(R.array.list_color_values);
```
Comment (jklein24) [inline app/src/main/java/org/connectbot/HostEditorFragment.java]: Should this TypedArray be recycled on detach?

Comment (khorimoto) [inline app/src/main/java/org/connectbot/HostEditorFragment.java]: Done.

**Pull request merged**